presto-product-tests/bin/locations.sh

```diff
@@ -2,3 +2,16 @@ SCRIPT_DIR=${BASH_SOURCE%/*}
 PRODUCT_TESTS_ROOT="${SCRIPT_DIR}/.."
 PROJECT_ROOT="${PRODUCT_TESTS_ROOT}/.."
 DOCKER_CONF_LOCATION="${PRODUCT_TESTS_ROOT}/conf/docker"
+
+# Yes, JDBC_DRIVER_JAR is intentionally set to a directory. 
+# This works better than leaving it empty or unset.
+# The open source Presto JDBC driver is included in the fat jar with the tests.
+JDBC_DRIVER_JAR="${PRODUCT_TESTS_ROOT}"
+TEMPTO_CONFIG_YAML="${PRODUCT_TESTS_ROOT}/conf/tempto/tempto-configuration-for-docker-default.yaml"
+TEMPTO_CONFIG_KERBEROS_YAML="${PRODUCT_TESTS_ROOT}/conf/tempto/tempto-configuration-for-docker-kerberos.yaml"
```

Comment:
Let's just have one TEMPTO_CONFIG_YAML variable, since tempto can only be run with one config. Let's define the variable value in each profile's `compose.sh` file, so that it's defined when using the compose.sh scripts standalone. Let's make it overridable using env vars.